cmd/manager/main.go

```diff
@@ -161,9 +175,48 @@ func main() {
 			},
 		}
 	}
+
+	// http/2 should be disabled due to its vulnerabilities. More specifically,
+	// disabling http/2 will prevent it from being vulnerable to the HTTP/2 Stream
+	// Cancellation and Rapid Reset CVEs. For more information see:
+	// - https://github.com/advisories/GHSA-qppj-fm5r-hxr3
+	// - https://github.com/advisories/GHSA-4374-p667-p6c8
+	disableHTTP2 := func(c *tls.Config) {
+		setupLog.Info("disabling http/2")
+		c.NextProtos = []string{"http/1.1"}
+	}
+
+	tlsOpts = append(tlsOpts, disableHTTP2)
+
+	// Create a new certificate watcher to watch for changes in the certificate files.
+	// If the certificate files change, the certificate watcher will reload the certificate
+	// and key files and update the TLS configuration.
+	cw, err := certwatcher.New(certFile, keyFile)
```

Comment:
@tmshort to speed it up I might need your help.
See that all works fine until we try to use certwatcher, see that e2e tests before the last commit and this one fail in the e2e tests: https://github.com/operator-framework/operator-controller/actions/runs/12032812266/job/33545503759?pr=1475